src/pages/People.tsx

```diff
@@ -0,0 +1,67 @@
+import { useParams } from 'react-router-dom';
+import { PeopleItem } from '../components/PeopleItem';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+
+export const People: React.FC<{
+  peopleFromServer: Person[] | undefined;
+  errorMessage: string | undefined;
+}> = ({ peopleFromServer, errorMessage }) => {
+  const { slug = '' } = useParams();
+
+  if (peopleFromServer) {
```

Comment:
empty array will be true too, so it can be removed. Create `isLoading` state and change it when you make a call to api to true, if success => false.
Then use ternary opearator
isLoading ? <loading> : <...your page>

Something like that

Reply:
Created